### PR TITLE
feat:Add The Countchar solution

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]
all good

**Review by davidrobert99 — COMMENTED** [review]
Where is the file with the solution?

**Comment by zainabdnaya** [comment]
> Where is the file with the solution?

The Solution file is added

File: solutions/countchar.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package student
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply solutions/countchar.go]: done

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**